featurebyte/query_graph/node/function.py

```diff
@@ -223,6 +234,16 @@ def _derive_sdk_code(
             )
             function_parameters.append(func_param_val)
 
-        function_parameters = [str(arg) for arg in function_parameters]
-        expression = f"{self.parameters.function_name}({', '.join(function_parameters)})"
-        return [], ExpressionStr(expression)
+        out_var_name = var_name_generator.generate_variable_name(
+            node_output_category=operation_structure.output_category,
+            node_output_type=operation_structure.output_type,
+            node_name=self.name,
+        )
+        expression = get_object_class_from_function_call(
+            self.parameters.name,
+            *function_parameters,
+            module_path="featurebyte",
+            class_name="UDF",
+        )
+        statements: List[StatementT] = [(out_var_name, expression)]
+        return statements, out_var_name
```

Comment:
This is a temporary solution. We need to use an explicit method to construct UDF function
(https://featurebyte.atlassian.net/browse/DEV-1878).

Current approach will have issue if the following happens:
1. User create a global UDF and then create a feature using the global UDF.
2. User create a local UDF with the same name as the global one (the method in UDF accessor will be overridden by the local one).
3. User save the previously created feature (using global UDF). This will cause error as the feature definition execution will use the local UDF. The final generated graph will be different from the original graph.